azure-pipelines/netcore-deploy-template.yml

```diff
@@ -0,0 +1,24 @@
+parameters:
+  - name: 'azureSubscription'
+    default: 'devlevel3-dev (a797242d-ce60-41aa-98ca-2e22d5221cfe)'
```

Comment:
not really sure if we should have this subscription data here, maybe @martin-callejas-jalafund can confirm?

Reply:
yeah, my bad here bc it's been previously merged

@paulo-teran-jalafund please replace with this default value: `devlevel3-dev-conn`